plugin-catalog/src/components/plugins/InstalledList.tsx

```diff
@@ -17,53 +17,124 @@ interface Plugin {
 
 export interface PurePluginInstalledListProps {
   installedPlugins: Plugin[] | null;
+  otherInstalledPlugins: any[] | null;
   error: string | null;
 }
 
-export function PurePluginInstalledList({ installedPlugins, error }: PurePluginInstalledListProps) {
+export function PurePluginInstalledList({
+  installedPlugins,
+  otherInstalledPlugins,
+  error,
+}: PurePluginInstalledListProps) {
   return (
     <SectionBox title="Installed" textAlign="center" paddingTop={2}>
       {error ? (
         <Typography>{`Error loading Installed plugins: ${error}`}</Typography>
       ) : (
-        <SimpleTable
-          columns={[
-            {
-              label: 'Name',
-              getter: plugin => (
-                <Box>
-                  <Link
-                    routeName="/plugin-catalog/:repoName/:pluginName"
-                    params={{ repoName: plugin.repoName, pluginName: plugin.folderName }}
-                  >
-                    {plugin.pluginTitle}
-                  </Link>
-                </Box>
-              ),
-            },
-            {
-              label: 'Version',
-              getter: plugin => plugin.pluginVersion,
-            },
-            {
-              label: 'Repo',
-              getter: plugin => plugin.repoName,
-            },
-            {
-              label: 'Author',
-              getter: plugin => plugin.author,
-            },
-          ]}
-          emptyMessage="No plugins installed"
-          data={installedPlugins || []}
-        />
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: 2,
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 2,
+              alignItems: 'start',
+            }}
+          >
+            <Typography variant="h6" components="h2">
+              Installed from the Plugin Catalog
```

Comment:
The title of the section is already "Installed". So here we could just have `Plugins from the Catalog`.